editor/src/clj/editor/slice9.clj

```diff
@@ -108,8 +109,8 @@
 
 (defn vertex-data
   [{:keys [width height tex-coords] :as _frame} size slice9 pivot]
-  (let [^double texture-width (or width (first size) 1.0)
-        ^double texture-height (or height (second size) 1.0)
+  (let [texture-width (max 1.0 (double (or width (first size) 0.0)))
+        texture-height (max 1.0 (double (or height (second size) 0.0)))
```

Comment:
First issue: If the manually specified `size` was zero when untextured, we'd produce infinity values resulting from divide-by-zero.